app/scripts/filters/util.js

```diff
@@ -215,6 +215,28 @@ angular.module('openshiftConsole')
         return isSecure ? 'https://' : 'http://';
     };
   })
+  .filter('isLink', function() {
+    // Uses same regular expression for link detection as the angular linky filter, but just says whether something is a link or not
+    // adds start and end of line to the regexp to test if the entire string is a link
+    // See https://github.com/angular/angular.js/blob/master/src/ngSanitize/filter/linky.js
+    var LINKY_URL_REGEXP =
+        /^((ftp|https?):\/\/|(www\.)|(mailto:)?[A-Za-z0-9._%+-]+@)\S*[^\s.;,(){}<>"\u201d\u2019]$/i;
+    return function(link) {
+      if (!link) {
+        return link;
+      }
+      return LINKY_URL_REGEXP.test(link);
```

Comment:
That actually causes something like `git://git.eclipse.org/r/lyo/org.eclipse.lyo.ldp`   to be turned into a link